lib/node.js

```diff
@@ -336,19 +336,24 @@ Node.prototype.getInfo = function()
 	console.time('Got info');
 
 	try {
-		this.info.coinbase = web3.eth.coinbase;
 		this.info.node = web3.version.node;
 		this.info.net = web3.version.network;
 		this.info.protocol = web3.toDecimal(web3.version.ethereum);
 		this.info.api = web3.version.api;
 
 		console.timeEnd('Got info');
-		console.info(this.info);
 
 		return true;
 	}
 	catch (err) {
+		console.error("Couldn't get version ");
+
+	}
+	try{
```

Comment:
Don't forget about space between curly braces, make all code like 'try - catch' block above 
Also remove unnecessary space

keep you code clean

Reply:
Fixed issu